plugins/lang-plan-a/src/main/java/org/elasticsearch/plan/a/PlanALexer.g4

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
@jdconrad lets make this `src/main/antlr`. we only have to change the paths in the ant regeneration to match.